src/main/java/net/tylers1066/movecraftcannons/listener/TranslationListener.java

```diff
@@ -23,12 +23,12 @@ public void translateListener(CraftTranslateEvent e) {
             return;
 
         Craft craft = e.getCraft();
-        if (!(craft instanceof PlayerCraft))
+        if (!(craft instanceof PilotedCraft))
```

Comment:
We'll want to support translation of subcrafts as well, since we can easily future proof this against that case.